setup.py

```diff
@@ -86,6 +86,7 @@ def get_git_sha() -> str:
         "flask-appbuilder>=4.3.11, <5.0.0",
         "flask-caching>=2.1.0, <3",
         "flask-compress>=1.13, <2.0",
+        "flask-cors>=4.0.0",
```

Comment:
@butuzov 
```
![image](https://github.com/apache/superset/assets/36619387/afbf6060-82eb-4d9c-b761-bcc12d2c1b08)
```
Can try to downgrade flask-cors==3.0.10 to avoid python dependencies issue.

Reply:
@weikhor you'll also likely have to recompile the frozen dependencies used for CI using `pip-compile-multi`. Please refer to `CONTRIBUTING.md`.

Reply:
flask-cors is considered an extra requirement

https://github.com/apache/superset/blob/master/setup.py#L148